Gemfile

```diff
@@ -48,6 +48,10 @@ gem 'rails_admin', '~> 1.2'
 # gem 'cancancan', '~>2.0'
 
 gem 'sidekiq'
+gem 'sidekiq-scheduler'
+gem 'sidekiq-unique-jobs'
+gem 'sinatra', require: nil
```

Comment:
Curious why

Reply:
I was just playing with sinatra gem which required these . Ill remove sinatra and update it :)
Sorry for late reply